media-video/amdgpu-pro-amf/amdgpu-pro-amf-1.4.34.1787253.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit unpacker
+
+MY_PV_HIGH=$(ver_cut 1)
+MY_PV_MIDDLE=$(ver_cut 2)
+MY_PV_LOW=$(ver_cut 3)
+MY_PV_REV=$(ver_cut 4)
+
+MY_PV="${MY_PV_HIGH}.${MY_PV_MIDDLE}.${MY_PV_LOW}"
+MY_PV_FULL="${MY_PV}-${MY_PV_REV}"
+
+MY_PN="amf-amdgpu-pro"
+MY_PN_ENC="libamdenc-amdgpu-pro"
+
+INTERNAL_VER="6.1.3"
+UBUNTU_VER="22.04"
+
+DESCRIPTION="AMD's closed source Advanced Media Framework (AMF) driver for AMD GPUs"
+HOMEPAGE="https://www.amd.com/en/support"
+
+URI_PREFIX="repo.radeon.com/amdgpu/${INTERNAL_VER}/ubuntu/pool/proprietary"
+
+SRC_URI="
+	https://${URI_PREFIX}/a/${MY_PN}/${MY_PN}_${MY_PV_FULL}.${UBUNTU_VER}_amd64.deb -> ${P}.deb
+	https://${URI_PREFIX}/liba/${MY_PN_ENC}/${MY_PN_ENC}_1.0-${MY_PV_REV}.${UBUNTU_VER}_amd64.deb -> ${P}-enc.deb
+"
+
+S="${WORKDIR}"
+
+LICENSE="AMD-GPU-PRO-EULA"
+SLOT="0"
+KEYWORDS="-* amd64"
+
+IUSE="+radv pro video_cards_amdgpu"
```

Comment:
I do wonder what use is `video_carts_amdgpu` here, it seems pretty useless.

Reply:
It's the same use case as in [this discussion we've had](https://github.com/gentoo/gentoo/pull/24660#discussion_r835733081)

Reply:
You should write a comment on top of IUSE/REQUIRED_USE explaining it so I don't have to ask everytime :)